keadm/cmd/keadm/app/cmd/helm/installer.go

```diff
@@ -381,8 +383,15 @@ func (cu *KubeCloudHelmInstTool) handleProfile(profileValue string) error {
 			profileValue = "v" + profileValue
 		}
 
-		cu.Sets = append(cu.Sets, fmt.Sprintf("%s=%s", "cloudCore.image.tag", profileValue))
-		cu.Sets = append(cu.Sets, fmt.Sprintf("%s=%s", "iptablesManager.image.tag", profileValue))
+		if !SetsContainSubstring(cu.Sets, "cloudCore.image.tag") {
```

Comment:
the behavior is changed vs before, it is right `!SetsContainSubstring`?

Reply:
yes, without this, cu.Sets will contain multiple duplicated `cloudCore.image.tag`, one is here `--profile version=v1.10.0`, and will also contain one if we specify in cmd line `--set cloudCore.image.tag=v1.10.0`

Reply:
What is the priority of these two parameters？if user build their own cloudcore image with a different tag, how to  achieve this?

Reply:
--set has a higher priority, If we don't use `--set` to set a image tag, we can use `--profile version=v1.10.0` to specify one uniform image tag(this tag will apply for all images, including cloudcore, iptables-manager, and controller-manager)

for example, if we want to use v1.10.0 cloudcore image and v1.11.0 controller-manager, we can use the below commd
```
_output/local/bin/keadm init --advertise-address=10.0.0.115 --profile version=v1.10.0 --set controllerManager.enable=true --set controllerManager.image.tag=v1.11.0
```

Reply:
got it, thanks for your explain